[Listview]: Minor Text correction under styles

List View - backgroundColor -> Background colour widget 

### What is the expected behaviour?
Similar to other text styles, rename  backgroundColor -> Background colour

### What is the current behaviour?
Under styles tab in Listview widget, Background colour is written as backgroundColor (Refer the screenshot)

### How to reproduce the issue?

1. Visit the [link](https://app.tooljet.com/) and create a new application.
2. Now drag and drop Listview widget and inspect its properties
3. Under styles tab correct the text similar to other texts

### Screenshots or Screencast
<img width="1440" alt="Screenshot 2022-09-06 at 10 27 42 AM" src="https://user-images.githubusercontent.com/32157641/188550446-27d2f7dc-0048-4d29-81cc-82f5e81d9240.png">

I think I've fixed this but having some issues running the tests. I've followed the documentation [here](https://docs.tooljet.com/docs/contributing-guide/setup/docker) to install a local instance. 

If I run `docker-compose run --rm server npm run test`

```
npm ERR! Missing script: "test"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-09-07T09_52_35_060Z-debug.log
ERROR: 1
```

When I run

`docker run server --rm npm run`

to get a list of script options I get the following list:

```
Scripts available in tooljet@1.18.0 via `npm run-script`:
  prebuild:plugins
    npm run install:plugins
  install:plugins
    npm --prefix plugins install
  build:plugins
    npm --prefix plugins run build
  prebuild:plugins:prod
    npm run install:plugins
  build:plugins:prod
    NODE_ENV=production npm run build:plugins
  postbuild:plugins:prod
    npm --prefix plugins prune --production
  prebuild:frontend
    npm --prefix frontend install --production=false
  build:frontend
    NODE_ENV=production npm --prefix frontend run build
  postbuild:frontend
    npm --prefix frontend prune --production
  prebuild:server
    npm --prefix server install --production=false
  build:server
    NODE_ENV=production npm --prefix server run build
  postbuild:server
    npm --prefix server prune --production
  build
    npm run build:plugins:prod && npm run build:frontend && npm run build:server
  start:prod
    npm --prefix server run start:prod
  db:create
    npm --prefix server run db:create
  db:create:prod
    npm --prefix server run db:create:prod
  db:migrate
    npm --prefix server run db:migrate
  db:migrate:prod
    npm --prefix server run db:migrate:prod
  db:seed
    npm --prefix server run db:seed
  db:seed:prod
    npm --prefix server run db:seed:prod
  db:setup
    npm --prefix server run db:setup
  db:setup:prod
    npm --prefix server run db:setup:prod
  db:reset
    npm --prefix server run db:reset
  db:drop
    npm --prefix server run db:drop
  deploy
    cp -a frontend/build/. public/
  heroku-postbuild
    ./heroku-postbuild.sh
  prepare
    husky install
```

I believe these scrips are from the outer package.json file in the workdir (app) created within the docker container. The correct package.json file is within /app/server. I'm not sure how to access this. 

I've tried `docker-compose run --rm server -w /server npm run test` but not working.

--- a/frontend/src/Editor/WidgetManager/widgetConfig.js
+++ b/frontend/src/Editor/WidgetManager/widgetConfig.js
@@ -3355,7 +3355,13 @@ export const widgets = [
       onRowClicked: { displayName: 'Row clicked' },
     },
     styles: {
-      backgroundColor: { type: 'color' },
+      backgroundColor: {
+        type: 'color',
+        displayName: 'Background color',
+        validation: {
+          schema: { type: 'string' },
+        },
+      },
       visibility: {
         type: 'toggle',
         displayName: 'Visibility',

